backend/django_core/static_src/components/page/explore/Index.svelte

```diff
@@ -0,0 +1,220 @@
+            title: "Time Range",
+            class: "hidden flex-col md:gap-[0.35vw]",
+            value: "",
+            selected_items: []
+        },
+        genres: {
+            title: "Genres",
+            class: "md:flex flex-col md:gap-[0.35vw]",
+            value: "",
+            items: {
+                action: "Action",
+                adventure: "Adventure",
+                hentai: "Hentai",
+                romance: "Romance"
+            },
+            selected_items: []
+        },
+        year: {
+            title: "Year",
+            class: "md:flex flex-col md:gap-[0.35vw]",
+            value: "",
+            items: {
+                2023: "2023",
+                2022: "2022",
+                2021: "2021",
+                2020: "2020"
+            },
+            selected_items: []
+        },
+        season: {
+            title: "Season",
+            class: "md:flex flex-col md:gap-[0.35vw]",
+            value: "",
+            items: {
+                winter: "Winter",
+                spring: "Spring",
+                summer: "Summer",
+                fall: "Fall"
+            },
+            selected_items: []
+        },
+        format: {
+            title: "Format",
+            class: "hidden md:flex flex-col md:gap-[0.35vw]",
+            value: "",
+            items: {
+                tv_show: "TV Show",
+                movie: "Movie"
+            },
+            selected_items: []
+        },
+        airing_status: {
+            title: "Airing Status",
+            class: "hidden flex-col md:gap-[0.35vw]",
+            value: "",
+            selected_items: []
+        },
+        sort_by: {
+            title: "Sort by",
+            class: "hidden flex-col md:gap-[0.35vw]",
+            value: "",
+            selected_items: []
+        }
+    };
+
+    // Functions
+    const update_selected_items = (key: string, selected_item: [string, string]) => {
+        let filter_option = filter_options_mapping[key];
+        let is_selected = filter_option.selected_items?.some((item) => item[0] === selected_item[0]);
+
+        if (is_selected) {
+            filter_option.selected_items = filter_option.selected_items?.filter((item) => item[0] !== selected_item[0]);
+        } else {
+            filter_option.selected_items = [...filter_option.selected_items, selected_item];
+        }
+
+        // update filer_options_mapping
+        filter_options_mapping[key] = filter_option;
+    },
+    clear_selected_items = (key: string) => {
+        // update filter_options_mapping
+        filter_options_mapping[key].selected_items = [];
+    };
+</script>
+
+<section class="mt-20 flex flex-col p-5 md:mt-0 md:gap-[1.5vw] md:pb-[2.5vw] md:pl-[1.5vw] md:pr-[3.75vw] md:pt-0">
+    <div class="flex flex-col gap-2 md:gap-[0.5vw]">
+        <span class="text-2xl font-bold leading-none md:text-[2vw]">
+            Anime <span class="text-warning">Explore</span>
+        </span>
```

Comment:
`<div>` then `<span/>` 

not

 `<span/>`  in  `<span/>`

Reply:
Okay, but its quite common to nest `span` tags.
Changed anyway :)